docker-install.sh

```
@@ -0,0 +1,59 @@
+echo "Docker Engine 설치 중..." >> $LOG_FILE
+sudo apt-get update >> $LOG_FILE 2>&1
+sudo apt-get install -y docker-ce docker-ce-cli containerd.io >> $LOG_FILE 2>&1
+
+# Docker 서비스 시작 및 활성화
+echo "Docker 서비스 시작 및 활성화 중..." >> $LOG_FILE
+sudo systemctl start docker >> $LOG_FILE 2>&1
+sudo systemctl enable docker >> $LOG_FILE 2>&1
+
+# 현재 사용자를 docker 그룹에 추가
+echo "현재 사용자를 docker 그룹에 추가 중..." >> $LOG_FILE
+sudo usermod -aG docker $USER >> $LOG_FILE 2>&1
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)